### PR TITLE
Gives the issues api the ability to utilize request blocks

### PR DESCRIPTION
**Pull request by discorick**
I.E 

```ruby
gh.issues(10) do |request|
  request.headers["Accept"] = "super-fancy/prevew-header"
end
```

### CODE REVIEW

File: ghee.gemspec [hunk 1]
```diff
@@ -7,7 +7,7 @@ Gem::Specification.new do |s|
   s.version     = Ghee::VERSION
   s.authors     = ["Ryan Rauh","Jonathan Hoyt"]
   s.email       = ["rauh.ryan@gmail.com"]
-  s.homepage    = "http://github.com/rauhryan/ghee"
+  s.homepage    = "http://github.com/huboard/ghee"
```
Comment (drusellers) [inline ghee.gemspec]: :+1:

**Comment by drusellers** [comment]
As a non-ruby dev - looks legit.

File: ghee.gemspec (hunk 1 above)
Comment (dahlbyk) [inline ghee.gemspec]: https

**Comment by dahlbyk** [comment]
Are there any other proxies that can/should be made to accept blocks while we're here?

**Comment by discorick** [comment]
> Are there any other proxies that can/should be made to accept blocks while we're here?

Was hoping for minimal impact on this one. The answer is yes in probably quite a few places, can we defer to a larger ghee upgrade ? Theres also new endpoints and things we are going to want to implement.

**Comment by dahlbyk** [comment]
> can we defer to a larger ghee upgrade ?

:+1:

**Pull request merged**